/dvs/p4/build/sw/devtools/Agora/Rel/QuadD_Main/QuadD/Common/ProtobufComm/Client/ClientProxy.cpp(389): Throw in function QuadDProtobufComm::Endpoint QuadDProtobufComm::Client::ClientProxy::GetLocalEndpoint() const
Dynamic exception type: boost::wrapexcept<QuadDCommon::RuntimeException>
std::exception::what: RuntimeException
[QuadDCommon::tag_message*] = Local Endpoint is not available.

srun: error: ault43: task 2: Exited with exit code 1
srun: got SIGCONT
slurmstepd: error: *** JOB 512035 ON ault43 CANCELLED AT 2024-09-29T00:21:23 ***
srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
srun: forcing job termination
slurmstepd: error: *** STEP 512035.1 ON ault43 CANCELLED AT 2024-09-29T00:21:23 ***
